include/z64player.h

```diff
@@ -217,39 +290,40 @@ typedef enum PlayerDoorType {
 } PlayerDoorType;
 
 typedef enum PlayerAnimType {
-    /* 0 */ PLAYER_ANIMTYPE_0,
+    /* 0 */ PLAYER_ANIMTYPE_0, // DEFAULT
```

Comment:
```suggestion
    /* 0 */ PLAYER_ANIMTYPE_0, // Default
```
Probably don't need all caps, o could just name it `PLAYER_ANIMTYPE_DEFAULT` if you think that's okay